### PR TITLE
fix test 9431 and 

### PR DESCRIPTION
**Pull request by leleOFA**
## Short description

Include a summary of the changes.

## List of changes proposed in this pull request

- rename step
- delete control of date useless
- deleted .yaml logic in other tests

## How to test

./mvnw clean test -Dcucumber.filter.tags=@TA_PFaggiuntaDelegaGiaPresente        

- [ ] Does this PR require a manual test in AWS CodeBuild?

### CODE REVIEW

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/common/BackgroundTest.java [hunk 1]
```diff
@@ -24,6 +27,18 @@ public class BackgroundTest {
...
+    private static Map<String, String> datiPersonaFisica;
+    static
+    {
+        datiPersonaFisica = new HashMap<>();
+        datiPersonaFisica.put("nome", "Lucrezia");
+        datiPersonaFisica.put("cognome", "Borgia");
+        datiPersonaFisica.put("codiceFiscale", "BRGLRZ80D58H501Q");
+        datiPersonaFisica.put("ente", "Comune di Verona");
+        datiPersonaFisica.put("ragioneSociale", "Lucrezia Borgia");
+    }
```
Comment (afus0) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/common/BackgroundTest.java]: just put it without the **brackets** and the **static**

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/DeleghePagoPATest.java [hunk 2]
```diff
@@ -122,14 +120,14 @@ public void nellaSezioneDelegheSiVisualizzaLaDelegaInStatoDiAttesaDiConferma() {
...
-    @And("Nella sezione Deleghe si verifica sia presente una delega")
+    /*@And("Nella sezione Deleghe si verifica sia presente una delega")
     public void nellaSezioneDelegheSiVerificaSiaPresenteUnaDelega() {
         logger.info("Si controlla che sia presente una delega");
         BackgroundTest backgroundTest = new BackgroundTest();
         if (!this.deleghePage.siVisualizzaUnaDelega()) {
             backgroundTest.aggiuntaNuovaDelegaPF();
         }
-    }
+    }*/
```
Comment (afus0) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/DeleghePagoPATest.java]: if unused just remove it

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/DeleghePagoPATest.java [hunk 3]
```diff
@@ -280,6 +278,7 @@ public void siControllaCheLaDelegaHaLoStatoAttiva(String dpFile) {
         logger.info("Si controlla che la delega abbia lo stato Attiva");
         this.deleghe = this.dataPopulation.readDataPopulation(dpFile + ".yaml");
         this.leTueDelegheSection.controlloStatoAttiva(deleghe.get("name").toString(), deleghe.get("familyName").toString());
+        this.leTueDelegheSection.controlloStatoAttiva(deleghe.get("name").toString(), deleghe.get("familyName").toString());
```
Comment (afus0) [inline src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/DeleghePagoPATest.java]: you do not need to `.toString()`, the `.get()` return the type of the value declared on the hashmap

**Review by afus0 — CHANGES_REQUESTED** [review]

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/DeleghePagoPATest.java (hunk 2 above)
Comment (leleOFA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/DeleghePagoPATest.java]: done

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/DeleghePagoPATest.java (hunk 3 above)
Comment (leleOFA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/DeleghePagoPATest.java]: image

File: src/test/java/it/pn/frontend/e2e/stepDefinitions/common/BackgroundTest.java (hunk 1 above)
Comment (leleOFA) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/common/BackgroundTest.java]: i tried more clean insert data, but this is the best method to insert data in an HashMap which is accepted by the function that accepts that parameter

Comment (afus0) [reply src/test/java/it/pn/frontend/e2e/stepDefinitions/common/BackgroundTest.java]: Create the map instance upon background test creation like so:

```
private Map<String, String> datiPersonaFisica;

public BackgroundTest(){
        datiPersonaFisica = new HashMap<>();
        datiPersonaFisica.put("nome", "Lucrezia");
        datiPersonaFisica.put("cognome", "Borgia");
        datiPersonaFisica.put("codiceFiscale", "BRGLRZ80D58H501Q");
        datiPersonaFisica.put("ente", "Comune di Verona");
        datiPersonaFisica.put("ragioneSociale", "Lucrezia Borgia");
    }
```

**Review by afus0 — CHANGES_REQUESTED** [review]

**Review by afus0 — APPROVED** [review]

**Pull request merged**